packages/odf/components/kms-config/utils.tsx

```diff
@@ -131,6 +134,21 @@ const generateConfigMapPatch = (
   };
 };
 
+const generateThalesSecret = (kms: ThalesConfig) => ({
+  apiVersion: SecretModel.apiVersion,
+  kind: SecretModel.kind,
+  metadata: {
+    name: `thales-kmip-kms-${Math.random().toString(36).substring(7)}`,
```

Comment:
```suggestion
    name: `thales-kmip-kms-${getRandomChars(7)}`,
```
Can we please refactor this logic out for others as well?